packages/kbn-plugin-generator/sao_template/template/package_template.json

```diff
@@ -13,8 +13,8 @@
     "es": "node ../../scripts/es",
     "lint": "eslint .",
     "start": "plugin-helpers start",
-    "test:server": "plugin-helpers test:server",
-    "test:browser": "plugin-helpers test:browser",
+    "test:mocha": "plugin-helpers test:server",
```

Comment:
I find this redirection a bit confusing since we are calling these by the underlying test runner, where in plugin-helpers we are calling them by what they are testing.

Curious your thoughts on why the switch here.